website/docs/d/container_app.html.markdown

```diff
@@ -93,6 +93,8 @@ A `volume` block supports the following:
 
 * `storage_type` - The type of storage volume. Possible values include `AzureFile` and `EmptyDir`. Defaults to `EmptyDir`.
 
+* `mount_options` - Mount options used while mounting the AzureFile. Must be a comma-separated string.
```

Comment:
In the data source docs we don't usually provide information on the type of input the property accepts, since these are all computed attributes and not settable by the user.
```suggestion
* `mount_options` - Mount options used while mounting the AzureFile.
```